packages/snaps-controllers/src/snaps/SnapController.test.ts

```diff
@@ -2593,7 +2595,7 @@ describe('SnapController', () => {
         .mockImplementationOnce(async () => {
           const manifest: SnapManifest = {
             ...getSnapManifest(),
-            version: '1.1.0',
+            version: '1.1.0' as SemVerVersion,
```

Comment:
This could be moved to a constant on top of the file.

Reply:
Not possible in this case. Tests that use specific versions instead of creation utils, do it because they're testing version/updates functionality.